be/src/Project/Services/Ordering/Ordering.Api/Validators/CheckoutBasketServiceModelValidator.cs

```diff
@@ -14,36 +15,29 @@ public CheckoutBasketServiceModelValidator()
             this.RuleFor(x => x.SellerId).NotNull().NotEmpty();
             this.RuleFor(x => x).Must(y =>
             {
-                if (y.Items == null)
+                if (y.Items.OrEmptyIfNull().Any())
```

Comment:
**suggestion:** very nice. There should be also a check for a hasCustomOrder and if  moreInfo isn't empty when the list of items is null or empty.